class AppErrorException(Exception):
    pass

class MailErrorException(Exception):
    pass